test/integer_infinite_numberpicker_test.dart

```diff
@@ -97,6 +108,8 @@ Future<NumberPicker> _testNumberPicker(
     int scrollBy,
     int step = 1,
     int expectedValue,
+    int zeroPadding = 1,
+    List<String> expectedDisplayValues,
```

Comment:
Could you explain to me if `expectedDisplayValues` is necessary?
I'd like to keep it as simple as possible :)